packages/lsp/src/ws-connection/ws-connection.ts

```diff
@@ -239,9 +251,26 @@ export class LspWsConnection implements ILspConnection {
    */
   protected serverCapabilities: protocol.ServerCapabilities;
 
+  /**
+   * The connection is connected?
+   */
   protected _isConnected = false;
+
+  /**
+   * The connection is initialized?
+   */
   protected _isInitialized = false;
 
+  /**
+   * Array of LSP callback disposables, it is used to
+   * clear the callbacks when the connection is disposed.
+   */
+  protected _disposables: Array<protocol.Disposable> = [];
```

Comment:
`protocol.Disposable` is coming for lsp isn't it?

If not could we switch to `DisposableSet`?
https://github.com/jupyterlab/lumino/blob/5faf82fe4e04d2a9d71923281c45233ca5bf6c8a/packages/disposable/src/index.ts#L115

Reply:
Yes, they are disposables of the `lsp` package, not the `lumino`